src/nvofapi_cuda.cpp

```diff
@@ -0,0 +1,58 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
Lets leave this info out here since this file is a community contribution as part of this PR .